Microsoft.Recognizers.Text.Number/French/Parsers/FrenchNumberParserConfiguration.cs

```diff
@@ -0,0 +1,250 @@
+﻿using Microsoft.Recognizers.Text.Number.Parsers;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using System.Text;
+
+namespace Microsoft.Recognizers.Text.Number.French.Parsers
+{
+    public class FrenchNumberParserConfiguration : INumberParserConfiguration
+    {
+        public FrenchNumberParserConfiguration(): this(new CultureInfo("fr-FR")) { }
```

Comment:
Can you update this to use the culture from https://github.com/Microsoft/Recognizers-Text/blob/master/Microsoft.Recognizers.Text.Number/Utilities/Culture.cs? Add French there, and update the constructor as [here](https://github.com/Microsoft/Recognizers-Text/blob/master/Microsoft.Recognizers.Text.Number/English/Parsers/EnglishNumberParserConfiguration.cs#L13)